webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -375,10 +375,10 @@ export default {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
     },
     isRedactorSpace() {
```

Comment:
I suggest to put canUpdateNewVisibilty
return !editMode && this.news.id !== '' && this.currentSpace && this.currentSpace.redactorsCount > 0;